docker/README.rst

```diff
@@ -297,13 +296,15 @@ the RunestoneServer directory do:
 
 .. code-block:: bash
 
-    scripts/dshell
+    docker/docker_tools.py shell
 
 
 Remember that the folder under web2py applications/runestone is bound to your host,
 so **do not edit files from inside the container** otherwise they will have a change
 in permissions on the host.
 
+To run Python-based program, you must first activate a virtual environment: use ``source /srv/venv/bin/activate`` or ``cd /srv/BookServer; poetry shell`` as appropriate.
```

Comment:
Do the updates to README explain this? If not, I'm open to any suggestions!

Reply:
is there a reason we can't use venv for both? That would be easier for novice contributors.

Reply:
Poetry runs a bit differently -- while activating Poetry's venv would work, doing pip installs and other venv operations would produce some confusion. I think it's best to start the right way, even if it's a bit more up-front work.

Reply:
I would actually prefer to go the other way because poetry does so much more for me!  I can make a release of bookserver with `poetry build` followed by `poetry publish` as opposed to a custom script I have to run every time I release RunestoneComponents.  But this all takes time and even though it IS progress, sometimes progress is painful.  I’m sorry your students have suffered from some of this but there is not a github repo anywhere that does not have some issues with the main branch especially when undergoing MAJOR transitions.

I’m happy that we now have 2/3’s of Runestone in a place where we can make releases to pypi and therefore have tags on the branch that indicate these release points.  One day we will hopefully get the instructor app to that same point!